feat: add new plugin fuse

bertux opened the pull request:
# Relates to

#3002

<!-- This risks section must be filled out before the final review and merge. -->

# Risks

Low, only adding one new plugin dedicated to one EVM chain.

# Background

## What does this PR do?

This PR integrates a plugin intended for the Fuse blockchain, which includes onchain actions:
* **Transfer tokens** between addresses on Fuse
* **Create a new ERC20 token** using the ERC20Factory contract

## What kind of change is this?

Features (non-breaking change which adds functionality)

# Documentation changes needed?

My changes do not require a change to the project documentation.

<!-- Please show how you tested the PR. This will really help if the PR needs to be retested and probably help the PR get merged quicker. -->

# Testing

## How we have tested the PR?

By integrating plugin-fuse in eliza-starter and starting the new actions by discussing with the Telegram bot. Our integration test is visible at https://github.com/bertux/eliza-starter/tree/plugin-fuse-integration

## Where should a reviewer start?

By looking at the automated tests of `plugin-fuse`

## Detailed testing steps

1. Add `FUSE_PRIVATE_KEY` to `.env`
2. Ensure you have a wallet funded with Fuse tokens for testing
3. Run tests to validate:
   * Token transfers between addresses
   * Creating a new ERC20 token
4. Execute `createToken` action to generate a new ERC20 token
5. Execute `transfer` action to send tokens to another wallet

## Discord username

bertux.celo

cryptokoh reviewed (commented):
Looking forward to using this asap

bertux commented on agent/src/index.ts:
```diff
@@ -1224,6 +1225,8 @@ export async function createAgent(
                 : null,
+            getSecret(character, "FUSE_PRIVATE_KEY") ? fusePlugin : null,
+        ].filter(Boolean),
             getSecret(character, "ZERO_EX_API_KEY") ? zxPlugin : null,
```
thanks I've fixed it.

guspan-tanadi committed:
chore(packages/clients): include section links

docs(packages/adapters): section links
chore(packages/plugins): implementation links Allora

tcm390 committed:
Merge pull request #3041 from sky-coderay/develop

fix: spelling issue

shakkernerd committed:
Merge branch 'develop' into fix-plugin-ethstorage

shakkernerd committed:
Merge pull request #3039 from AIFlowML/fix-plugin-ethstorage

fix(lint): plugin-ethstorage

shakkernerd committed:
Merge branch 'develop' into fix-plugin-evm

shakkernerd committed:
Merge pull request #3038 from AIFlowML/fix-plugin-evm

fix(lint): plugin-evm

shakkernerd committed:
Merge branch 'develop' into fix-plugin-flow

shakkernerd committed:
Merge pull request #3036 from AIFlowML/fix-plugin-flow

fix(lint): plugin-flow

shakkernerd committed:
Merge branch 'develop' into fit-plugin-football

shakkernerd committed:
Merge pull request #3035 from AIFlowML/fit-plugin-football

fix(lint): plugin-football

shakkernerd committed:
Merge branch 'develop' into fix-plugin-form

shakkernerd committed:
Merge pull request #3033 from AIFlowML/fix-plugin-form

fix(lint): plugin-form

shakkernerd committed:
Merge branch 'develop' into fix-plugin-fuel

shakkernerd committed:
Merge pull request #3028 from AIFlowML/fix-plugin-fuel

fix(lint): plugin-fuel

shakkernerd committed:
Merge branch 'develop' into fix-plugin-gelato

shakkernerd committed:
Merge pull request #3027 from AIFlowML/fix-plugin-gelato

fix(lint): plugin-gelato

shakkernerd committed:
Merge branch 'develop' into fix-plugin-genlayer

shakkernerd committed:
Merge pull request #3025 from AIFlowML/fix-plugin-genlayer

fix(lint): plugin-genlayer

shakkernerd committed:
Merge branch 'develop' into fir-plugin-giphy

shakkernerd committed:
Merge pull request #3024 from AIFlowML/fir-plugin-giphy

fix(lint): plugin-giphy

shakkernerd committed:
Merge branch 'develop' into fix-plugin-goat

shakkernerd committed:
Merge pull request #3019 from AIFlowML/fix-plugin-goat

fix(lint): plugin-goat

shakkernerd committed:
Merge branch 'develop' into fix-plugin-gitcoin-passport

shakkernerd committed:
Merge pull request #3022 from AIFlowML/fix-plugin-gitcoin-passport

fix(lint): plugin-gitcoin-passport

shakkernerd committed:
Merge branch 'develop' into fix-plugin-gitbook

shakkernerd committed:
Merge pull request #3023 from AIFlowML/fix-plugin-gitbook

fix(lint): plugin-gitbook

tobySolutions committed:
feat: add  Fleek Eliza deployment to Readme_CN (#3048)

* feat: add  Fleek Eliza deployment to readme

* Update README_CN.md

tcm390 committed:
correct auto post condition

tcm390 committed:
use room id for memory

tcm390 committed:
Merge branch 'develop' into tcm-telegram-duplicated-id

tcm390 committed:
correct regex

bertux committed:
Update transfer.ts

for chain config

tcm390 committed:
Merge pull request #3054 from elizaOS/tcm-correct-regex

fix: correct regex

shakkernerd committed:
Merge branch 'develop' into tcm-telegram-duplicated-id

tcm390 committed:
Merge pull request #3053 from elizaOS/tcm-telegram-duplicated-id

fix: Message id collision in Telegram Client

tcm390 committed:
add imageVisionModelProvider

tcm390 committed:
Merge branch 'develop' into tcm-imageVisionModelProvider

tcm390 committed:
Merge pull request #3056 from elizaOS/tcm-imageVisionModelProvider

fix: ImageVisionModelProvider Not Applied in Runtime for Image Description Service

tcm390 committed:
handle unspport image provider

tcm390 committed:
Merge pull request #3057 from elizaOS/tcm-image-model-handling

fix: handle unsupported image provider

odilitime committed:
Merge pull request #3012 from v1xingyue/plugin-sui-swap

feat: use Aggregator swap sui tokens

bertux commented on packages/plugin-fuse/src/actions/transfer.ts:
```diff
@@ -0,0 +1,167 @@
+                data: params.data as Hex,
+                kzg: undefined,
+                chain: undefined,
+            });
```
update done and automatic tests passed.

YoungPhlo committed:
docs: minor formatting for consistency

bertux committed:
Merge branch 'develop' into plugin-fuse

YoungPhlo committed:
Merge branch 'develop' into docs/community-streams

YoungPhlo committed:
docs: minor formatting for consistency

ai16z-demirix committed:
plugin-bootstrap: package json

ai16z-demirix committed:
plugin-bootstrap: vitest config

ai16z-demirix committed:
plugin-bootstrap: continue tests

ai16z-demirix committed:
plugin-bootstrap: fact tests

AIFlowML committed:
fixed all the issues but the file types.ts

ai16z-demirix committed:
plugin-bootstrap: goal tests

odilitime committed:
Merge branch 'develop' into fix-plugin-di

AIFlowML committed:
Fixed all the issues

AIFlowML committed:
Merge branch 'develop' into fix-plugin-dexscreener

ai16z-demirix committed:
plugin-chainbase: package json

ai16z-demirix committed:
plugin-chainbase: vitest config

ai16z-demirix committed:
plugin-chainbase: query data tests

ai16z-demirix committed:
plugin-chainbase: retrieve token balance tests

ai16z-demirix committed:
plugin-chainbase: libs/chainbase tests

AIFlowML committed:
Fixed all the issues

AIFlowML committed:
Merge branch 'develop' into fix-plugin-devin

ai16z-demirix committed:
Revert "plugin-bootstrap: goal tests"

This reverts commit 19bce8107ba95910433f21b66a1c0d951db6647f.

ai16z-demirix committed:
Revert "plugin-bootstrap: continue tests"

This reverts commit 3450d1f8938f6466cbf2aab64f4b4dd42fd46656.

ai16z-demirix committed:
Revert "plugin-bootstrap: fact tests"

This reverts commit df5590bfeb432d866e48463e0bdc9ccb97e86a3e.

ai16z-demirix committed:
Revert "plugin-bootstrap: package json"

This reverts commit 46062f04950f42854025b651e8311e887a78ef03.

ai16z-demirix committed:
Revert "plugin-bootstrap: vitest config"

This reverts commit b407f925c68e470a0958aee3e6bbe1d79f05560d.

AIFlowML committed:
Fixed all the issues

AIFlowML committed:
Merge branch 'develop' into fix-plugin-dcap

odilitime committed:
Merge pull request #3070 from AIFlowML/fix-plugin-devin

fix: plugin-devin lint

AIFlowML committed:
Fixed all issues

Commit:
add arabic translation

AIFlowML committed:
Merge branch 'develop' into fix-plugin-cronoszkevm

AIFlowML committed:
Fixed almost all issues bt could not taouch the types.ts

Commit:
Add Arabic translation to README language links

AIFlowML committed:
Merge branch 'develop' into fix-plugin-dexscreener

AIFlowML committed:
Fixed all the issues but 1 any

AIFlowML committed:
Merge branch 'develop' into fix-plugin-binance

odilitime committed:
Merge pull request #3089 from AIFlowML/fix-plugin-binance

fix: plugin-binance lint

AIFlowML committed:
fix: plugin-birdeye (#3088)

* Fixed almost all issues bt could not taouch the types.ts

* Fixed all the issues

AIFlowML committed:
Fixed almost all the issues (#3087)

Co-authored-by: Odilitime <janesmith@airmail.cc>

AIFlowML committed:
fix: plugin-Chainbase lint (#3085)

* Fixed almost all issues bt could not taouch the types.ts

* Fixed all the issue but the template file.

AIFlowML committed:
fix: plugin-coingecko lint (#3084)

* Fixed almost all issues bt could not taouch the types.ts

* Fixed all the issues

AIFlowML committed:
fix: plugin-coinmarketcap lint (#3082)

* Fixed almost all issues bt could not taouch the types.ts

* Fixed few minor issues

AIFlowML committed:
Fixed almost all issues bt could not taouch the types.ts (#3080)

AIFlowML committed:
Fixed all issues but a file (#3077)

AIFlowML committed:
Fixed all the issues (#3076)

AIFlowML committed:
Fixed almost all the issues (#3086)

Co-authored-by: Odilitime <janesmith@airmail.cc>

odilitime committed:
Merge pull request #3026 from PhyByte/develop

fix: Rename file with typo in MVX plugin

odilitime committed:
Merge branch 'develop' into develop

odilitime committed:
Merge pull request #3081 from EmanHerawy/develop

docs: Add Arabic translation for README

odilitime committed:
Merge pull request #3075 from AIFlowML/fix-plugin-cronoszkevm

fix: plugin-cronoszkevm lint

odilitime committed:
Merge pull request #3074 from AIFlowML/fix-plugin-dcap

fix: plugin-dcap lint

AIFlowML committed:
Fixed all issues, i have an shadow error on packages/plugin-b2/src/providers/index.ts line 103 it appear and disapper (#3091)

AIFlowML committed:
Fixed all the issues (#3073)

odilitime committed:
Merge branch 'develop' into fix-plugin-dexscreener

odilitime committed:
Merge pull request #3069 from AIFlowML/fix-plugin-dexscreener

fix: plugin-dexscreener lint

btspoony committed:
fix: the erroneous corrections caused by biome. (#3052)

- fix the sample serviceType is undefined
- fix erroneous corrections in di plugin caused by biome.
- add Parameter Decorators support in biome

odilitime committed:
Merge branch 'develop' into fix-plugin-di

odilitime committed:
Merge pull request #3066 from AIFlowML/fix-plugin-di

fix: fix-plugin-di

AIFlowML committed:
Fixed all the issues (#3092)

Co-authored-by: Odilitime <janesmith@airmail.cc>

AIFlowML committed:
Fixed all the errors (#3063)

AIFlowML committed:
Fixed all the issues (#3065)

Co-authored-by: Odilitime <janesmith@airmail.cc>

AIFlowML committed:
Fixed all typings and wrong variables (#3061)

MichaelDeng03 committed:
feat: 3049 add demo api access coingecko (#3050)

* Update getPrice.ts to use headerKey in coin price request.

* Update getMarkets.ts to check api config for api headerkey

AIFlowML committed:
Fixed all the issues and added biome (#3044)

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

AIFlowML committed:
fix: plugin-avail  (#3094)

* Fixed all the issues

* fix: add biome configuration and fix linting issues in plugin-avail

odilitime committed:
Merge pull request #3005 from guspan-tanadi/adapterslinks

docs(packages/adapters): navigate section links

john-xina-p88 committed:
chore: readme

john-xina-p88 committed:
feat: sequentially cancel orders

john-xina-p88 committed:
Merge branch 'develop' into feature/plugin-desk-exchange

john-xina-p88 committed:
Merge pull request #1 from HMXOrg/feature/plugin-desk-exchange

feat: plugin-desk-exchange

samarth30 committed:
fix: fixed a type issue due to which build was failing

samarth30 committed:
fix: fixed a type issue due to which build was failing

AIFlowML committed:
fix: add biome configuration and fix linting issues in plugin-autonome

tcm390 committed:
Merge pull request #3097 from elizaOS/fix/plugin-email-automation

fix: fixed build error in plugin-email-automation

AIFlowML committed:
Great code i just added the Biome

YoungPhlo committed:
Merge branch 'develop' into docs/community-streams

john-xina-p88 committed:
feat: use bigint to prevent overflow

john-xina-p88 committed:
Merge branch 'develop' into develop

YoungPhlo committed:
docs: Add "What Did You Get Done This Week? 11" notes

AIFlowML committed:
Fixed all the issues this is the commit to keep for arthers

AIFlowML committed:
Fixed all the issues in Aptos and aded Biome

AIFlowML committed:
Merge branch 'develop' into fix-plugin-asterai-clean

AIFlowML committed:
Fixed the initial error in wallet and multiple other typings. I left 2 because not possibler to restore them fast. (#3010)

Co-authored-by: tcm390 <60634884+tcm390@users.noreply.github.com>

tcm390 committed:
Merge branch 'develop' into fix-plugin-asterai-clean

tcm390 committed:
Merge pull request #3101 from AIFlowML/fix-plugin-asterai-clean

fix: plugin-asterai

tcm390 committed:
Merge branch 'develop' into fix-plugin-autonome-clean

tcm390 committed:
Merge pull request #3098 from AIFlowML/fix-plugin-autonome-clean

fix: plugin-autonome-v1

tcm390 committed:
Merge branch 'develop' into fix-plugin-arthera-biome-clean

tcm390 committed:
Merge pull request #3102 from AIFlowML/fix-plugin-arthera-biome-clean

fix: plugin-arthera-biome

tcm390 committed:
Merge branch 'develop' into fix-plugin-aptos

tcm390 committed:
Merge pull request #3104 from AIFlowML/fix-plugin-aptos

fix: plugin-aptos

tcm390 committed:
Merge branch 'develop' into docs/community-streams

tcm390 committed:
Merge pull request #3103 from YoungPhlo/docs/community-streams

docs: Add "What Did You Get Done This Week? 11" notes

AIFlowML committed:
Investigated and added the peoper ether to the configuration it was missing and fixed all the code

AIFlowML committed:
Merge branch 'develop' into fix-plugin-apro

samarth30 committed:
Merge branch 'develop' into plugin-fuse

jonathangus committed:
Update Dockerfile

vipocenka committed:
typo index.ts

vipocenka committed:
fix typo api.ts

vipocenka committed:
fix typo GoplusSecurityService.ts

vipocenka committed:
fix typo README.md

maxime committed:
fix: fixing the error parsing json when an array is a value in a JSON object

maxime committed:
Merge branch 'develop' into bug/json_parsing_of_arrays_as_value

john-xina-p88 committed:
Merge branch 'develop' into develop

tcm390 committed:
fix scroll to bottom

ai16z-demirix committed:
Merge branch 'develop' into tests/plugin-chainbase

ai16z-demirix committed:
Update package.json

tcm390 committed:
Merge pull request #3113 from maxime/bug/json_parsing_of_arrays_as_value

fix: fixing the error parsing json when an array is a value in a JSON…

tcm390 committed:
Merge branch 'develop' into fix/typos

tcm390 committed:
Merge branch 'develop' into tcm-fix-client-scroll

wtfsayo committed:
updated gitignore + removed build files (#3117)

ai16z-demirix committed:
Merge branch 'develop' into tests/plugin-chainbase

bertux committed:
Merge branch 'develop' into plugin-fuse

shakkernerd committed:
Merge branch 'develop' into tcm-fix-client-scroll

tcm390 committed:
Merge pull request #3115 from elizaOS/tcm-fix-client-scroll

fix: client scroll to bottom

shakkernerd committed:
Merge branch 'develop' into fix-plugin-apro

shakkernerd committed:
Merge pull request #3106 from AIFlowML/fix-plugin-apro

fix: plugin-apro

ai16z-demirix committed:
Merge branch 'develop' into tests/plugin-chainbase

shakkernerd committed:
Merge pull request #3072 from ai16z-demirix/tests/plugin-chainbase

feat (chore): plugin-chainbase test config and coverage

ai16z-demirix committed:
feat (chore): plugin bittensor test config and coverage (#3064)

* plugin-bittensor: package json

* plugin-bittensor: vitest config

* plugin-bittensor: actions, sn34 test

* plugin-bittensor: fact tests

* Update fact.test.ts

* Update package.json

fixing conflict

* Update package.json

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

ai16z-demirix committed:
feat (chore): plugin-bootstrap: test config and test coverage (#3068)

* Update fact.test.ts

* Update goal.test.ts

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

shakkernerd committed:
Merge branch 'develop' into fix/typos

shakkernerd committed:
Merge branch 'develop' into docker-fix

shakkernerd committed:
Merge pull request #3110 from jonathangus/docker-fix

fix: docker build command by skipping eslint files

Commit:
chore(plugin-chainbase): add @biomejs/biome dev dependency

shakkernerd committed:
fix: export dexScreenerPlugin

shakkernerd committed:
Merge pull request #3120 from elizaOS/fix/plugin-dexscreener

fix: export dexScreenerPlugin

shakkernerd committed:
Merge branch 'develop' into fix/plugin-chainbase

shakkernerd committed:
Merge pull request #3118 from akugone/fix/plugin-chainbase

fix:chore(plugin-chainbase): add missing comma in package.json

alexpaden committed:
save imageUrls for outbound tweets/messages

interactions.ts handles inbound
utils.ts handles outbound

shakkernerd committed:
fix: remove duplicate litPlugin import

shakkernerd committed:
Merge pull request #3121 from elizaOS/fix/agent_duplicates

fix: remove duplicate litPlugin import

0xCardiE committed:
add provider, action and template for pools per network

0xCardiE committed:
change names

bertux committed:
Merge branch 'develop' into plugin-fuse

odilitime committed:
Merge pull request #2361 from elizaOS/develop

chore: dev => main 0.1.9

tcm390 committed:
remove duplicated plugins

john-xina-p88 committed:
Merge branch 'develop' into develop

vipocenka committed:
Update api.ts

tcm390 committed:
Merge branch 'develop' into fix/typos

tcm390 committed:
Merge pull request #3111 from vipocenka/fix/typos

fix: typos in multiple files

tcm390 committed:
Merge branch 'develop' into feature/twitter-messages-image-outbound

tcm390 committed:
Merge pull request #3122 from alexpaden/feature/twitter-messages-image-outbound

feat: save imageUrls for outbound tweets/messages

HashWarlock committed:
fix: upgrade openai and vercel ai packages to fix o1 errors

bertux committed:
fix: Enhance token creation examples and improve error messaging in createTokenAction (#6)

lalalune committed:
Merge pull request #3146 from elizaOS/fix/openai-o1-fix

fix: upgrade openai and vercel ai packages to fix o1 errors

strmfos committed:
Update chat_2024-12-07.md

strmfos committed:
Update chat_2024-10-27.md

AIFlowML committed:
Fixed all the issues and added biome

AIFlowML committed:
Merge branch 'develop' into fix-plugin-allors

0xCardiE committed:
Merge branch 'develop' into feat/coingecko_advanced_more_endpoints

AIFlowML committed:
Fixed multiple issues and aded biome.

AIFlowML committed:
Merge branch 'main' into fix-plugin-akash-v1

AIFlowML committed:
Fixed the typing and added biome

AIFlowML committed:
Merge branch 'main' into fix-plugin-agentkit

AIFlowML committed:
fixed all the issues but deployTokenAction.ts for the ANY and the result need probaly a refactoring.

AIFlowML committed:
Merge branch 'develop' into fix-plugin-abstract

AIFlowML committed:
Merge branch 'develop' into fix-plugin-agentkit

0xCardiE committed:
add new pools by network

AIFlowML committed:
Fixed all the issues and added biome

AIFlowML committed:
Merge branch 'develop' into fix-plugin-3g-gen

AIFlowML committed:
Fixed all the issues and added Biome

AIFlowML committed:
Merge branch 'develop' into fix-plugin-0x

0xCardiE committed:
add mappings from provider

AIFlowML committed:
Fixed all the issues and added biome. Only issues still in place in upload.ts i had to add an any to overcome the issue on the line 370 const flowContract = getFlowContract(runtime.getSetting(ZEROG_FLOW_ADDRESS), signer as any);

AIFlowML committed:
Merge branch 'develop' into fix-plugin-0g

AIFlowML committed:
Added biome to - plugin-zksync-era - plugin-zilliqa - plugin-zerion - plugin-whatsapp

AIFlowML committed:
Merge branch 'develop' into fix-multi-biome-01

AIFlowML committed:
Add Biome linting and formatting configuration to the following plugins:
- plugin-web-search
- plugin-video-generation
- plugin-udio
- plugin-twitter

shakkernerd committed:
Merge pull request #3180 from AIFlowML/fix-multi-biome-01

fix: multi-biome-01

shakkernerd committed:
Merge branch 'develop' into fix-plugin-0g

AIFlowML committed:
Merge branch 'develop' into fix-multi-biome-02

shakkernerd committed:
Merge pull request #3179 from AIFlowML/fix-plugin-0g

fix: plugin-0g

shakkernerd committed:
Merge branch 'develop' into fix-multi-biome-02

shakkernerd committed:
Merge pull request #3181 from AIFlowML/fix-multi-biome-02

fix: multi-biome-02

shakkernerd committed:
chore: remove multiple tsup package

shakkernerd committed:
Merge branch 'develop' into fix-plugin-0x

shakkernerd committed:
Merge pull request #3178 from AIFlowML/fix-plugin-0x

fix: plugin-0x

shakkernerd committed:
Merge branch 'develop' into fix-plugin-3g-gen

shakkernerd committed:
Merge pull request #3175 from AIFlowML/fix-plugin-3g-gen

fix: plugin-3g-generation

shakkernerd committed:
Merge branch 'develop' into fix-plugin-abstract

shakkernerd committed:
Merge pull request #3174 from AIFlowML/fix-plugin-abstract

fix: plugin-abstract

shakkernerd committed:
Merge branch 'develop' into fix-plugin-agentkit

shakkernerd committed:
Merge pull request #3172 from AIFlowML/fix-plugin-agentkit

fix: plugin-agentkit

shakkernerd committed:
Merge branch 'develop' into fix-plugin-akash-v1

shakkernerd committed:
Merge pull request #3171 from AIFlowML/fix-plugin-akash-v1

fix: plugin-akash

shakkernerd committed:
Merge branch 'develop' into fix-plugin-allors

shakkernerd committed:
Merge pull request #3169 from AIFlowML/fix-plugin-allors

fix: plugin-allora

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #3165 from comfsrt/main

docs: fix typos in .md

ai16z-demirix committed:
feat (chore): plugin-coinmarketcap (#3134)

* plugin-coinmarketcap: vitest config

* plugin-coinmarketcap: package json

* plugin-coinmarketcap: test setup

* plugin-coinmarketcap: get price tests

* plugin-coinmarketcap: get price service tests

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

ai16z-demirix committed:
feat (chore): plugin-coingecko test config and coverage (#3124)

* plugin-coingecko: package json

* plugin-coingecko: vitest config

* plugin-coingecko: get price tests

* plugin-coingecko: test setup

* plugin-coingecko: get trending tests

* plugin-coingecko: top gainers losers tests

* plugin-coingecko: get market tests

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

tcm390 committed:
Merge branch 'main' into tcm/remove-duplicated-plugins

tcm390 committed:
Merge branch 'develop' into tcm/remove-duplicated-plugins

tcm390 committed:
Merge pull request #3126 from elizaOS/tcm/remove-duplicated-plugins

fix: remove duplicated plugins

tcm390 committed:
update provider-utils

tcm390 committed:
Merge branch 'develop' into tcm-update-provider-utils

tcm390 committed:
Merge pull request #3189 from elizaOS/tcm-update-provider-utils

fix: update provider-utils

tcm390 committed:
extract attribute from raw text instead of normalized json

tcm390 committed:
Merge pull request #3190 from elizaOS/tcm-correct-extract

fix: extract attribute from raw text instead of normalized json

samarth30 committed:
Merge branch 'develop' into feat/coingecko_advanced_more_endpoints

john-xina-p88 committed:
Merge branch 'develop' into develop

samarth30 committed:
Merge pull request #3170 from 0xCardinalError/feat/coingecko_advanced_more_endpoints

feat: coingecko advanced -  various pools by network

samarth30 committed:
Merge pull request #3045 from edwin-finance/plugin-edwin

feat: Add edwin plugin to eliza

samarth30 committed:
Merge pull request #3096 from HMXOrg/develop

feat: plugin desk exchange

nilaysarma committed:
Update Twitter to X (Twitter)

Since Twitter is now rebranded to X, so I have changes made changes in the README file. It is now "X (Twitter)" and I kept the Twitter keyword since people still use the word Twitter.

tcm390 committed:
Merge branch 'develop' into 3196--Update-Twitter-to-X

tcm390 committed:
Merge pull request #3198 from nilaysarma/3196--Update-Twitter-to-X

docs: Update Twitter to X (Twitter)

bertux committed:
Merge branch 'develop' into plugin-fuse

odilitime commented:
Hello,

We are changing our plugin development strategy to be more scalable. We have moved the plugins out into their own repos and we're looking for people to either maintain those or own them on their own Github.

If you'd like to be a maintainer,  file an issue in the plugin repo and join our Discord https://discord.gg/elizaos to coordinate.

If you'd like to control the plugin on your own Github, please add an issue to the plugin repo pointing to your repo, and add a modification to the registry. Submit a PR to edit the registry here: https://github.com/elizaos-plugins/registry

Closing this PR for now. Let us know if you have any questions
